content/docs/setup/kubernetes/istio-cni-install/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Istio CNI Install Options
+description: Instructions for installing and using Istio with the Istio CNI plugin.
+weight: 70
+keywords: [kubernetes,cni]
+---
+
+The Istio CNI plugin implements the networking details for injecting the Istio sidecar proxy between application pods
+and the container networking infrastructure.  Starting with Istio 1.1, Using the Istio CNI plugin removes the
+requirement for the privileged, `NET_ADMIN` container in the Istio users' application pods.  The
+[Istio CNI plugin](https://github.com/istio/cni) replaces the functionality provided by the `istio-init`
+container injected into application pods' `initContainers` list.
+
+## Prerequisites
+
+1.  Kubernetes installed with the container runtime supporting CNI and `kubelets` configured
+    with the main CNI plugin enabled--`network-plugin=cni`.
+    *  This is enabled for the Kubernetes installation for IBM Cloud IKS, Azure AKS, and AWS EKS clusters.
+    *  For Google Cloud GKE clusters, the
+       [network-policy](https://cloud.google.com/kubernetes-engine/docs/how-to/network-policy) feature
+       must be enabled to have Kubernetes configured with `network-plugin=cni`.
+
+1.  Kubernetes installed with the [ServiceAccount admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#serviceaccount) enabled.
+    *  The Kubernetes documentation highly recommends this for all Kubernetes installations
+       where `ServiceAccounts` are utilized.
+
+## Installation steps
+
+The installation of the Istio CNI plugin is integrated with the [Istio Installation with Helm](/docs/setup/kubernetes/helm-install/) procedure via the addition of the setting `--set istio_cni.enabled=true`.
+
+{{< text bash >}}
```

Comment:
This doesn't render as you want in md

Reply:
I think this is okay, as this content will be rendered correctly on the istio.io website. See 
https://deploy-preview-2902--preliminary-istio.netlify.com/docs/setup/kubernetes/istio-cni-install/#installation-steps.

Reply:
This seems the way folks are doing this in the istio.io docs

Reply:
Ok - wasn't rendering correctly in my local view,  but if good on main it is fine with me.